src/Pages/PeoplePage.tsx

```diff
@@ -0,0 +1,83 @@
+import { useCallback, useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoaded, setIsLoaded] = useState<boolean>(false);
+  const [isError, setIsError] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isLoaded, setIsLoaded] = useState(false);
  const [isError, setIsError] = useState(false);
```

Reply:
i know) but in case many letters of code was better to write type of states obviously